src/Git/GitStatusListCtrl.h

```diff
@@ -722,6 +724,11 @@ class CGitStatusListCtrl :
 
 	void DeleteSelectedFiles();
 
+	void MoveToChangelist(const CString& name);

```

Comment:
Move this to the other changelist related methods?

Reply:
sure, what line do you think is the best?